pkg/pop/poptoken.go

```diff
@@ -0,0 +1,11 @@
+package pop
+
+import "github.com/Azure/kubelogin/pkg/internal/pop"
+
+func GetSwPoPKey() (*SwKey, error) {
```

Comment:
Let's get rid of this and let the caller to instantiate this.

Reply:
I think the issue is that the caller does not have access to the internal *SwKey object. I tried, but I could only make it work using this function to convert it from the *pop.SwKey.